cmp/grid/impl/ColumnHeader.js

```diff
@@ -121,24 +126,50 @@ class LocalModel {
     }
 
     onClick = (e) => {
```

Comment:
should we be more explicit about what handlers we expect to be registered/running in a mobile context, and what in a desktop context?  Just looking at this code,  you might think there is some risk of these things getting called twice.